Tests/varLib/varLib_test.py

```diff
@@ -1,8 +1,9 @@
 from __future__ import print_function, division, absolute_import
-from fontTools.misc.py23 import *
```

Comment:
Please don't remove this line . All files should start with the above two lines.

Reply:
Right